source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -0,0 +1,566 @@
+
+#. Disabling after install
+
+   To disable both of the telemetry daemons, run the following command:
+
+   .. code-block:: bash
+
+      sudo telemctl stop
+
+#. Opt in to telemetry
+
+   To opt-in to the telemetry services, simply enter the opt-in command and start the service:
+
+   .. code-block:: bash
+
+      sudo telemctl opt-in
+
+   This removes the :file:`/etc/telemetrics/opt-out` file, if it exists, and starts the telemetry services.
+
+   .. note::
+
+      To opt-in but not immediately start telemetry services, you will need to run the command :command:`sudo telemctl stop` after the :command:`opt-in` command is entered. Once you are ready to start the service, enter the command :command:`sudo telemctl start`.
+
+#. Opt out of telemetry
+
+   To stop sending telemetrics data from your system, opt out of the telemetry service:
+
+   .. code-block:: bash
+
+      sudo telemctl opt-out
+
+   This creates the file :file:`/etc/telemetrics/opt-out` and stops the telemetry services.
+
+
+Saving Data Locally
+===================
+
+This example requires |CL| to be installed, and telemetry to be enabled on the system.
+
+To change how records are managed, we will be copying the default :file:`/usr/share/defaults/telemetrics/telemetrics.conf` file to :file:`/etc/telemetrics/telemetrics.conf` and editing it.  The changes in the :file:`/etc/telemetrics/telemetrics.conf` file will override the defaults in the :file:`/usr/share/defaults/telemetrics/telemetrics.conf` file. You may need ``root`` permissions to create and edit files in :file:`/etc`. For each example, and for any time you make changes to the configuration file, you will need to restart the client daemons to pick up the changes:
+
+.. code-block:: bash
+
+  sudo telemctl restart
+
+
+The :command:`telemctl journal` command gives you access to features and options of the telemetry journal to assist with system analytics and debug. :command:`telemctl journal` has a number of options to help filter records. Use :command:`-h` or :command:`--help` to view usage options.
+
+
+#. Keep local copy and send records to backend server
+
+   To keep a local copy of the telemetry record and also send it on to the backend server, we will need to change the :guilabel:`record_retention_enabled` configuration key value to :guilabel:`true`.
+
+
+#. Keep all records  -- don't send to backend server
+
+   To keep records on the system without sending them to a backend server, set the :guilabel:`record_server_delivery_enabled` key value to :guilabel:`false`.  Note that you will also need to ensure the the :guilabel:`record_retention_enabled` configuration key value is set to :guilabel:`true` or the system will not keep local copies.
```

Comment:
remove duplicate "the"..."Note that you will also need to ensure the ..."

Reply:
in next commit